module2/gift-certificate-system/service/src/test/resources/test-db.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why do you use different types of Spring configuration inside one project, Java class in main module and xml for tests? This could be good for educational purposes but it looks strange in general.

Reply:
In the task there was a point: Several configurations should be implemented. As I understand it, this applies to the Spring configuration? I am currently rewriting tests using Mock, and this configuration is redundant.